packages/encrypt-upload-client/README.md

```diff
@@ -41,11 +41,24 @@ const encryptedClient = await EncryptClient.create({
 })
 ```
 
+### Browser Usage
+
+For browser apps, use the `BrowserCryptoAdapter`:
+
+```js
+import { BrowserCryptoAdapter } from '@storacha/encrypt-upload-client/dist/crypto-adapters/browser-crypto-adapter.js'
```

Comment:
it's surprising to me you need the dist directory... usually our packages are setup to read from that folder directly for NPM distributions.

Reply:
My bad. You don't need the `dist`. Fixed.